dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
+        <button type="button" class="nav">Home</button>
+        <button type="button" class="nav">Projects</button>
+        <button type="button" class="nav">About Me</button>
+        <button type="button" class="nav">Contact</button>
+      </div>
+      <span class="perks">
+        Custom Designs · Marketing · Branding · And More
+      </span>
+      <div class="whois">
+        <h2> WHO IS SEEK-A-DESIGN?</h2>
+        </br>
+        <p>Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.</p>
+      </div>
+      <div class="seeking">
+        <h2>SEEKING-A-REVIEW?</h2>
+        </br>
+        <p>Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.</p>
```

Comment:
Similar to Line 25, except a bit harder...

Here, you can imagine that we have some sort of **persistent data store**; a place where we can store data long term. What data? Individual reviews that have been left by previous clients!

How many reviews do we have? Who knows. They will accumulate over time! When we get to the javascript part of this page, we'll have to make the length of this list **dynamic**; it will change based on the length of the array that contains all of the reviews.

For now, we can hard-code each review, but we should **code them as an un-ordered list of paragraphs**, rather than just a series of paragraphs.

To do this:

- Use the `<ul>` tag to create an unordered list.
- Use a bunch of `<li></li>` tags, nested inside the `<ul></ul>`.
- Inside each `<li>` tag, create a paragraph (`<p></p>`) containing the text for a single review.
- Go to your `style.css` file, and figure out how to remove the bullet points from your list.
- Continue styling the list to match the styling of the mockup.